lib/control/v1/transit.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const TransitProvider = require('../../providers/transit');
+
+function Transit(storage) {
+  return (req, res, next) => {
+    storage.lookup(req.params.token, req.body, TransitProvider)
+    .then((result) => {
+      // Check for both undefined and null. `!= null` handles both cases.
+      if (result != null && result.hasOwnProperty('plaintext')) {
+        result.plaintext = Buffer.from(result.plaintext, 'base64').toString();
```

Comment:
I ran into this too when I was poking at this PR. If we're pinning to `lts/argon` we'll be fine.